CalDAVClient.php

```diff
@@ -176,7 +186,7 @@ function getctag() {
     }
 
     function updateEvent($url, $etag, $data) {
-        $ch = $this->init_curl_request($this->url . '/' . $url);
+        $ch = $this->init_curl_request("{$this->url}/$url");
```

Comment:
oh, nice, I was not aware we could have this kind of variable extrapolation in php, that's pretty convenient! 👍